cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/CVRUtilities.java

```diff
@@ -406,7 +408,7 @@ public String getVariantAsHgvs(MutationRecord record) {
     }
 
     public String convertWhitespace(String s) {
-        return s.replaceAll("^[\\t|\\n|\\r]+", "").replaceAll("[\\t|\\n|\\r]+$", "").replaceAll("[\\t|\\n|\\r]+", " ");
```

Comment:
I noticed here that we used to replace tabs and newlines with empty space, and I have replace it with a single space.  @n1zea144 is that OK?